docs/Stdlib/python/Tutorials/data_processing.md

```diff
@@ -1,78 +1,314 @@
 # How to process your data
 
-1. Load data into a table
+## Load or create a Table
 
-    ```python
-    # read a csv file
-    table_csv = Table.from_csv("path_to_csv_file.csv")
+First we need data to work with. There are different ways to create the data.
+The data will be stored in a [Table][safeds.data.tabular.Table], a [Row][safeds.data.tabular.Row] or a [Column][safeds.data.tabular.Column].
 
-    # or a json file
-    table_json = Table.from_json("path_to_json_file.json")
-    ```
+A Table can be created with python Iterables or you can load [csv][safeds.data.tabular.Table.from_csv] or [json][safeds.data.tabular.Table.from_json] files into a Table.
 
-1. Extract a row from a table
+```python
+from safeds.data.tabular import Table
 
-    ```python
-    # provide the index of the row
-    row = table.get_row(0)
-    ```
+table = Table({"column1": [3, 4, 8, 6, 5],
+               "column2": [2, 2, 1, 6, 3],
+               "column3": [1, 1, 1, 1, 1]})
 
-1. Extract a column from a table
+table_from_csv = Table.from_csv("path/to/your/csv/file.csv")
 
-    ```python
-    # provide the name of the column
-    column = table.get_column("TheColumnName")
-    ```
+table_from_json = Table.from_json("path/to/your/json/file.json")
+```
 
-1. Combine a list of rows to a table
+This table has the following output:
 
-    ```python
-    # make sure the rows have the same columns
-    table = Table.from_rows([row1, row2])
-    ```
+| column1 | column2 | column3 |
+| ------- | ------- | ------- |
+| 3       | 2       | 1       |
+| 4       | 2       | 1       |
+| 8       | 1       | 1       |
+| 6       | 6       | 1       |
+| 5       | 3       | 1       |
 
-1. Combine a list of columns to a table
+## Create Rows and Columns
 
-    ```python
-    # make sure the columns have the same amount of rows
-    table = Table.from_columns([column1, column2])
-    ```
+Columns and Rows can be created with python Iterables or extracted from an existing Table.
```

Comment:
```suggestion
Columns and rows can be created with python Iterables or extracted from an existing table.
```

Reply:
I think the capitalization might be useful because the tutorial is referencing the Row and Table Classes